apps/explorer-e2e/src/integration/asset.cy.js

```diff
@@ -0,0 +1,20 @@
+import '../support/common.functions';
+
+context('Asset page', function () {
+  describe('Verify elements on page', function () {
+    const assetsNavigation = 'a[href="/assets"]';
+    const assetHeader = '[data-testid="asset-header"]';
+
+    it('Assets page is displayed', function () {
+      cy.visit('/');
```

Comment:
suggestion: cy.visit should be placed in before hook

Reply:
I didn't put this in a before hook because `cy.common_switch_to_mobile_and_click_toggle();` already goes to homepage so only 1 tests needs cy.visit